Assets/Scripts/PlayerControl.cs

```diff
@@ -109,21 +183,22 @@ IEnumerator SmoothMove()
             float distanceCovered = (Time.time - startTime) * moveSpeed;
             float fractionOfJourney = distanceCovered / journeyLength;
 
-            // Move the player using lerp
-            transform.position = Vector3.Lerp(startPosition, endPosition, fractionOfJourney);
+            // Update player position
+            Vector3 newPosition = Vector3.Lerp(startPosition, endPosition, fractionOfJourney);
+            transform.position = newPosition;
 
             // If we're very close to the target, snap to it
             if (Vector3.Distance(transform.position, endPosition) < 0.01f)
             {
                 transform.position = endPosition;
             }
-
+            // Update lightstick positions to maintain relative positioning
+            UpdateLightstickPositions(endPosition);
```

Comment:
```suggestion
            UpdateLightstickPositions(transform.position);
```
이렇게 변경해야 합니다.
endPosition으로 하면, 플레이어는 아직 다 안움직였는데 lightStick들만 미리 이동을 완료해있게 되겠네요.